tests/tests.py

```diff
@@ -350,10 +350,8 @@ def test_channels_list_moderator(self):
         me = self.rocket.me().json()
         channels_add_moderator = self.rocket.channels_add_moderator(
```

Comment:
Usually testing if the API call returned success is a good idea. This way this check `local variable 'channels_add_moderator' is assigned to but never used` will be gone.

Reply:
Sorry, didn't see that comment. I think I should leave out adding/removing moderators before testing the list moderators API call. It should succeed even without any moderators in place, and my setup code just convolutes the test.
Give me a second...